justfile

```diff
@@ -306,6 +316,11 @@ destroy platform=default_platform:
         ;;
         "AKS-PEER-SNP")
             nix run -L .#terraform -- -chdir=infra/azure-peerpods destroy
+
+            # Clean-up cached image ids.
+            rm ${CONTRAST_CACHE_DIR}/*.image-id || true
+
+            nix run -L .#scripts.destroy-coco-aks -- --name="$azure_resource_group"
```

Comment:
We do that because we want to destroy the resource group named `$azure_resource_group`. Tthis resource group contains the image gallery. If it doesn't exist yet, we create this resource group shortly before uploading the image: https://github.com/edgelesssys/contrast/blob/105ffe01ef0a2191849d6bdaf360b3b141e40f7f/packages/scripts.nix#L60-L63

Reply:
I think we shouldn't share the aks resource group to also upload images there. I want to be able to destroy my aks setup without deleting the images that I use in a totally different peerpod cluster.

Also, I'd say it is better to inline both group creation and deletion directly into the justfile, so it's easier to understand what is happening. These are one-liners anyway.

Reply:
> I think we shouldn't share the aks resource group to also upload images there. I want to be able to destroy my aks setup without deleting the images that I use in a totally different peerpod cluster.

Do you want me to move the image gallery into the same resource group that contains the cluster or do you want the image gallery to be in another, new resource group?

Reply:
I'd say a separate group would be nice. If you aren't working on the image part, you might be happy with the same image for long time. On the other hand I'm not sure how to integrate that into the dev workflow, as it would require additional create/destroy steps for that. :thinking:

Reply:
That seems reasonable. Given that this PR has already dragged on for quite a while, are you fine with doing this in a follow-up PR?